docs/config-file/v2.rst

```diff
@@ -297,6 +297,16 @@ The path to the Conda environment file, relative to the root of the project.
 :Type: ``path``
 :Required: ``true``
 
+.. note::
+
+   With ``anaconda-project>=0.8.4``can use an `Anaconda Project`_ ``anaconda-project.yaml`` file as a 
+   Conda environment file by changing ``packages:`` to the alias ``dependencies:``.  This should work for 
+   `versions >=0.8.4 <https://github.com/Anaconda-Platform/anaconda-project/issues/194>`_,
```

Comment:
I think there is no need to link the GitHub issue, since this is now described in the Anaconda Project docs: https://anaconda-project.readthedocs.io/en/stable/user-guide/reference.html#commands-and-requirements